doc/common/chapter00.md

```diff
@@ -215,14 +215,14 @@ The ETSI NFV ISG is very closely related to the Anuket project, in that it is a
 <p align="center"><b>Figure 5:</b> Scope ETSI NFV.</p>
 
 <a name="3.4.2"></a>
-### Relationship between Anuket projects and OVP
+### Relationship between Anuket projects and AAP
```

Comment:
Need to add AAP to the Abbreviations

Reply:
Thanks for pointing it out. We created a new issue for that: #2613